src/_pytest/skipping.py

```diff
@@ -291,7 +291,7 @@ def pytest_runtest_makereport(item: Item, call: CallInfo[None]):
             else:
                 rep.outcome = "passed"
                 rep.wasxfail = xfailed.reason
-    elif (
+    if (
```

Comment:
I suggest adding a newline above to make it clear it's a different code section.

```suggestion

    if (
```